src/styles/index.scss

```diff
@@ -2,2 +1,61 @@
+@import './variables';
+
+.page {
   margin: 0;
+  height: 100vh;
+  display: flex;
+  align-items: center;
+  justify-content: center;
+}
+
+.calendar {
+  display: flex;
+  flex-wrap: wrap;
+  gap: $days-gap;
+  width: ($calendar-item-size + $calendar-gap) * $calendar-item-columns;
+
+  &__day {
+    box-sizing: border-box;
+    position: relative;
+    padding: 10px;
+    display: flex;
+    align-items: center;
+    justify-content: center;
+    background-color: $calendar-item-bg-color;
```

Comment:
The variable `$calendar-item-bg-color` is used here but not defined in the provided code. Ensure that it is defined in the imported `variables` file or elsewhere in your code.